wave_matching/include/wave/matching/matcher.hpp

```diff
@@ -1,10 +1,17 @@
 #ifndef WAVE_MATCHING_MATCHER_HPP
 #define WAVE_MATCHING_MATCHER_HPP
 
+#include <exception>
 #include <Eigen/Core>
 #include <Eigen/Geometry>
 
 namespace wave {
+// define exception
+class ConfigException : public __exception {
+    virtual const char* what() const throw() {
+        return "Failed to Load Matcher Config";
+    }
+};
```

Comment:
I looked at http://www.cplusplus.com/doc/tutorial/exceptions/ for the standard exceptions. The closest one on there is "runtime_error" which doesn't seem very helpful.